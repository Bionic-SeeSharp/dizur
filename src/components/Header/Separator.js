import styled from 'styled-components';

const Separator = styled.h6`
  color: white;
  padding-top: 8px;
`;

export default Separator;
