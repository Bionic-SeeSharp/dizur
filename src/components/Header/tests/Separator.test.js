import React from 'react';
import { shallow } from 'enzyme';

import Separator from '../Separator';

describe('<Separator />', () => {
  it('render a Separator', () => {
    const wrapper = shallow(<Separator />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render a h6 tag', () => {
    const renderedComponent = shallow(<Separator />);
    expect(renderedComponent.type()).toEqual('h6');
  });
  it('should render a h6 tag', () => {
    const renderedComponent = shallow(<Separator />);
    expect(renderedComponent.type()).toEqual('h6');
  });
  it('should render children', () => {
    const children = '|';
    const renderedComponent = shallow(<Separator>{children}</Separator>);
    expect(renderedComponent.contains(children)).toBe(true);
  });
});
