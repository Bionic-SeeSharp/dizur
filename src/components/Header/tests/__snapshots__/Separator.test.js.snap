// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> render a Separator 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <styled.h6 />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "sc-bdVaJa bOdaLV",
    },
    "ref": null,
    "rendered": null,
    "type": "h6",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "sc-bdVaJa bOdaLV",
      },
      "ref": null,
      "rendered": null,
      "type": "h6",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
